ext/intl/breakiterator/breakiterator_class.h

```diff
@@ -42,7 +42,7 @@ typedef struct {
 } BreakIterator_object;
 
 static inline BreakIterator_object *php_intl_breakiterator_fetch_object(zend_object *obj) {
-	return (BreakIterator_object *)((char*)(obj) - XtOffsetOf(BreakIterator_object, zo));
+	return reinterpret_cast<BreakIterator_object *>(reinterpret_cast<char *>(obj) - XtOffsetOf(BreakIterator_object, zo));
```

Comment:
You can either do a C and C++ version of just falling back to just C version to keep it simple . Same for `codepointiterator_internal.h`

Reply:
Just keep the C version, there's no point in duplicating the code just  to have a C/C++ version to keep an IDE happy, that's just more error-prone.